go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/bufbuild/buf
 go 1.19
 
 require (
+	buf.build/gen/go/bufbuild/protovalidate/protocolbuffers/go v1.31.0-20230802163732-1c33ebd9ecfa.1
```

Comment:
I think we've said that we didn't want the buf CLI directly depending on remote packages due to the circular dependency issue that can introduce w.r.t. bootstrapping the buf toolchain - @rodaine do you have any opinions here?

Reply:
My primary concern is the BSR depending on the BSR; less so concerned about the CLI. Right now, adding this dep to the CLI will currently make it a dependency of the BSR codebase.

As for these options, either has different tradeoffs in terms of complexity in order to avoid the circular deps through the BSR. Having the concrete types present will make the code easier to write and prevent runtime issues while the underlying PV types are still in a bit of flux.

Reply:
I think overall, it sounds like we should remove this dependency. Generating these deps within `private/gen` doesn't seem to be a major issue, unless I'm missing something here - let me know.